libs/dal/src/repositories/integration/integration.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { ProviderEnum } from '@notifire/shared';
+
+interface ICredentials {
+  apiKey: string;
+  secretKey: string;
+}
+
+export class IntegrationEntity {
+  _id?: string;
+
+  _applicationId: string;
+
+  _organizationId: string;
+
+  providerId: string;
+
+  channel: ProviderEnum;
```

Comment:
We already have an enum for a channel called: ChannelTypeEnum you can import it from @notifire/shared library

Reply:
Noted i will replace it.